sw/device/silicon_creator/mask_rom/boot_policy_ptrs.h

```diff
@@ -23,7 +23,7 @@ static_assert((TOP_EARLGREY_EFLASH_SIZE_BYTES % 2) == 0,
  *
  * @return Pointer to the manifest of the ROM_EXT image in slot A.
  */
-inline const manifest_t *romextimage_slot_a_manifest_ptr_get(void) {
+inline const manifest_t *boot_policy_manifest_a_get(void) {
```

Comment:
I'm not sure if these should be `volatile` because of the case you mentioned: bootstrap will occur before we access these and IIRC, we plan to reset the chip after bootstrap. @moidx WDYT? I can also create an issue if we would like to revisit this later.

Reply:
Should we make it volatile to ensure we get a fresh read on this call? My main concern is that we don't have the bootstrap implementation yet, so I am not sure if the attacker could leverage glitching the reset after bootstrap. I am ok if you want to track this in an issue.

Reply:
Created #7893.